src/Components/Header/Header.js

```diff
@@ -0,0 +1,18 @@
+import BrandLogo from "../../commonComponents/BrandLogo";
+import AddressBar from "./AddressBar";
+import { navConfig, addressConfig } from "./constants/config";
```

Comment:
- instead of importing constants here and sending them as props, you can directly import them in child components and use them there

Reply:
I think passing it as a prop makes sense, at least for the addressConfig because it can be dynamic. Ya like for navConfig I can keep it the child component itself because it doesn't changes dynamically yet. Again a debatable question, we should discuss it.